ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1488,6 +1488,11 @@ internal class TableViewCellAccessoryView: UIView {
         button.accessibilityLabel = "Accessibility.TableViewCell.MoreActions.Label".localized
         button.accessibilityHint = "Accessibility.TableViewCell.MoreActions.Hint".localized
         button.addTarget(self, action: #selector(handleOnAccessoryTapped), for: .touchUpInside)
+
+        if #available(iOS 13.4, *) {
+            button.isPointerInteractionEnabled = true
```

Comment:
Is this actually part of our design for table view cells? Note that Apple doesn't actually have these enabled for any table view cells in the system that I can find:

Tested Contacts and Settings, "Information" disclosure button, "Call" and "text" buttons in Contacts, and Switch accessory views.

Reply:
Basically, the code looks fine if we want to be different than the system on this particular interaction, but I'm not convinced that's intentional.

Reply:
The pointer interaction will only be enabled on out "..." button in the TableViewCell. 
This is what the designers have asked for. There's a Teams discussion about this, let me know if you want to join the conversation.
For this particular interaction, we are okay with being different from the system.